support-frontend/assets/pages/new-contributions-landing/contributionsLanding.scss

```diff
@@ -975,121 +960,105 @@ form {
   margin-top: $gu-v-spacing;
 }
 
-.gu-content--thefrontline {
+//***************************
+// Campaign styles:
+//***************************
+.gu-content--campaign-landing {
```

Comment:
for https://github.com/guardian/support-frontend/pull/1804 to work this modifier class has to be the campaign name, i.e. `.gu-content--toxicamerica`

Reply:
or else we can modify the way it works so it's always adding `--campaign-landing` as well, if it has a campaign code

Reply:
👍

Reply:
PR here: https://github.com/guardian/support-frontend/pull/1809